unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BuilderInWorldEntityHandler.cs

```diff
@@ -147,10 +147,15 @@ public void ReportTransform(bool forceReport = false)
     {
         foreach (DCLBuilderInWorldEntity entity in selectedEntities)
         {
-            if (!entity.HasMovedSinceLastReport() && !forceReport)
+            if (!entity.HasMovedSinceLastReport() &&
+                !entity.HasScaledSinceLastReport() &&
+                !entity.HasRotatedSinceLastReport() &&
```

Comment:
Do we need this grade of granularity? I mean , what do you think to unify this methods in `HasTransformChangeSinceLastReport`? (Same for report)

I am ok with both parts, but maybe this can made the code more readeable

Reply:
Yeah, we could unify all in only one function but, if you don't mind, I would like to keep it separated because I think it could be useful for some future requirements where we need to detect these specifics actions.